src/NBitcoin/Protocol/Versions.cs

```diff
@@ -65,6 +65,13 @@ public enum ProtocolVersion : uint
         /// </summary>
         WITNESS_VERSION = 70012,
 
+        /// <summary>
+        /// ! Communication between nodes with proven headers is possible after this version.
```

Comment:
Remove `! `

Reply:
I was wondering what was this ! for, copy/pasted from above